src/web3/saga.test.ts

```diff
@@ -1,4 +1,4 @@
-import { isValidChecksumAddress } from '@celo/utils/lib/address'
+import { isValidChecksumAddress } from '@ethereumjs/util'
```

Comment:
can we replace it with viem's [isAddress](https://viem.sh/docs/migration-guide#utilities-isaddress--getaddress-perform-checksum-validation)?
It can do the checksum too I think.
This way we can avoid pulling `@ethereumjs/util` if I'm not mistaken.